parsers.py

```diff
@@ -27,9 +28,12 @@ def __str__(self) -> str:
         return f"date: {self.date}, symbol: {self.symbol}, price: {self.price}"
 
 
-def read_broker_transactions(schwab_transactions_file: str) -> List[BrokerTransaction]:
+def read_broker_transactions(
+    schwab_transactions_file: str, trading212_transactions_folder: str
+) -> List[BrokerTransaction]:
     reader_transactions = [
         read_schwab_transactions(schwab_transactions_file),
+        read_trading212_transactions(trading212_transactions_folder),
     ]
```

Comment:
I think we shouldn't merge them. Otherwise you can end up with selling shares from one account on another account which will cause incorrect results. Let's chose which broken we want to use somewhere.

Reply:
The thing is that for tax calculations it should be merged. HMRC pool system needs to include all the shares you own regardless of where you obtained them. So if I want to calculate my gains I need to include the transactions from all the brokers. For example buying FOO in some broker and then selling FOO acquired two years ago in some other broker will count as Same day rule.

Reply:
Yeah, I think you're right. Ok, let's merge then.